frontend/src/libs/import-export/validate.js

```diff
@@ -8,7 +8,7 @@
  * @param {*} schema
  */
 export function validate(data, schema) {
-    const { requiredKeys } = schema;
+    const { requiredKeys, keys } = schema;
     for (const item of data) {
         for (const key of requiredKeys) {
             if (item[key] == null) {
```

Comment:
same as above.
`null == undefined` is true, `null === undefined` is false. This if statement would include the case where `item[key]` is undefined

Reply:
yes, and the purpose of this is to convert undefined to null.